src/components/datasets/ExtendedInformationPanel.vue

```diff
@@ -9,44 +10,43 @@ const props = defineProps({
 })
 
 const contactPoint = props.dataset.contact_point
-const accessRights = props.dataset.extras?.harvest?.['dct:accessRights']
-const provenance = (
-  props.dataset.extras?.harvest?.['dct:provenance'] || []
-).filter((p: string) => !!p)
+const dcatExtras = props.dataset.extras?.dcat
+const uri = props.dataset.harvest?.uri
 
-const hasExtendedInfo =
-  !!accessRights || !!contactPoint || provenance.length > 0
+const hasExtendedInfo = !!contactPoint || !!dcatExtras || !!uri
 </script>
 
 <template>
   <div
     v-if="hasExtendedInfo"
-    class="fr-py-3w fr-mb-3w border-bottom border-default-grey"
+    class="fr-pb-3w border-bottom border-default-grey"
   >
-    <h2 class="subtitle subtitle--uppercase">Informations étendues</h2>
-    <div class="fr-text--sm fr-m-0">
-      <div class="fr-grid-row fr-grid-row--gutters">
-        <div v-if="accessRights" class="fr-col-12 fr-col-sm-6 fr-col-md-4">
-          <h3 class="subtitle fr-mb-1v">Conditions d'accès et d'utilisation</h3>
-          <p class="fr-text--sm fr-m-0 text-mention-grey">
-            {{ accessRights }}
-          </p>
-        </div>
-        <div v-if="contactPoint" class="fr-col-12 fr-col-sm-6 fr-col-md-4">
-          <h3 class="subtitle fr-mb-1v">Point de contact</h3>
-          <p class="fr-text--sm fr-m-0 text-mention-grey">
-            <a :href="`mailto:${contactPoint.email}`">{{
-              contactPoint.name
-            }}</a>
-          </p>
-        </div>
-        <div v-if="provenance.length" class="fr-col-12 fr-col-sm-6 fr-col-md-4">
-          <h3 class="subtitle fr-mb-1v">Généalogie</h3>
-          <ul class="fr-text--sm fr-m-0 text-mention-grey">
-            <li v-for="p in provenance" :key="p">{{ p }}</li>
-          </ul>
-        </div>
+    <div class="metadata-list fr-text--sm fr-m-0">
+      <ExtendedInformationPanelItem
+        v-if="uri"
+        :items="[uri]"
+        title="Identifiant de ressource unique"
+      />
+      <ExtendedInformationPanelItem
+        :items="dcatExtras.accessRights"
+        title="Conditions d'accès et d'utilisation"
+      />
+      <ExtendedInformationPanelItem
+        :items="dcatExtras.provenance"
+        title="Généalogie"
+      />
+      <div v-if="contactPoint">
+        <h3 class="subtitle fr-mb-2v">Point de contact</h3>
+        <p class="fr-text--sm fr-m-0">
+          <a :href="`mailto:${contactPoint.email}`">{{ contactPoint.name }}</a>
+        </p>
       </div>
     </div>
   </div>
 </template>
+
+<style scoped>
+.metadata-list > *:last-child {
```

Comment:
pour info: ça marche sans l'étoile ⭐  

```suggestion
.metadata-list > :last-child {
```